components/Front/index.js

```diff
@@ -103,9 +103,9 @@ class Front extends Component {
 export default compose(
   withT,
   graphql(getDocument, {
-    options: () => ({
+    options: props => ({
       variables: {
-        path: '/',
+        path: props.path || '/',
```

Comment:
why not use `props.url.asPath.split('?')[0]` directly here? Would make it unnecessary to introduce a new path props and work just like the article page.